docs/install.rst

```diff
@@ -188,37 +195,28 @@ Building from Source
 Prerequisites
 -------------
 
-You will need a compiler suite and the development headers for Python and
-NumPy in order to build ``astropy``.
-
-If you are building the latest developer version rather than using a stable
-release, you will also need `Cython <https://cython.org/>`_ (v0.29.13 or later) and
-`jinja2 <https://jinja.palletsprojects.com/en/master/>`_ (v2.7 or later) installed. The
-released packages have the necessary C files packaged with them, and hence do
-not require Cython.
+You will need a compiler suite and the development headers for Python in order
+to build ``astropy``. You do not need to install any other specific build
+dependencies (such as `Cython <https://cython.org/>`_ or
+`jinja2 <https://jinja.palletsprojects.com/en/master/>`_) since these are
+declared in the ``pyproject.toml`` file and will be automatically installed into
+a temporary build environment by pip.
 
 Prerequisites for Linux
 -----------------------
 
 On Linux, using the package manager for your distribution will usually be the
 easiest route to making sure you have the prerequisites to build ``astropy``. In
-order to build from source, you will need the Python and Numpy development
-package for your Linux distribution, as well as a number of helper packages.
+order to build from source, you will need the Python development
+package for your Linux distribution, as well as pip.
 
 For Debian/Ubuntu::
 
-    sudo apt-get install python3-dev python3-numpy-dev python3-setuptools cython3 python3-jinja2 python3-pytest-astropy
```

Comment:
I just changed this section to be more `pip` agnostic - many people on distributions prefer not to create another distribution system, especially one that does not clean up (like `pip`) and does not work for all users. Can you leave this as is?

Reply:
I've changed this back for now